zetta_utils/augmentations/imgaug.py

```diff
@@ -114,13 +117,21 @@ def _ungroup_kwargs(
 @builder.register("imgaug_readproc")
 def imgaug_readproc(
     *args,  # the zetta_utils builder puts the layer/layerset as the first argument
+    targets: Iterable[str] | None = None,
```

Comment:
Sorry to ask, but could you add a comment to this please? Or (better) use something like

```
    """
    Read processor using imgaug.

    :param *args:
    :param ...
    """
```
Also not a fan of `*args` + `assert` instead of just using `data` (which makes it a little clearer that it conforms to the proc protocol), but that's fine.